crates/bevy_ecs/macros/src/fetch.rs

```diff
@@ -166,13 +166,19 @@ pub fn derive_world_query_impl(ast: DeriveInput) -> TokenStream {
 
         quote! {
             #derive_macro_call
+            #[doc = "Item for ["]
```

Comment:
"Automatically generated [`WorldQuery`] item type for [`Foo`], returned when iterating over query results."